app/src/androidTest/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/schedule/ScheduleParentFragmentTest.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.eventmanager.ui.eventShowcase.fragments.schedule;
+
+import android.content.Intent;
+import android.os.SystemClock;
+import android.support.test.espresso.contrib.DrawerActions;
+import android.support.test.espresso.contrib.NavigationViewActions;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.Gravity;
+import ch.epfl.sweng.eventmanager.R;
+import ch.epfl.sweng.eventmanager.ui.eventSelector.EventPickingActivity;
+import ch.epfl.sweng.eventmanager.ui.eventShowcase.EventShowcaseActivity;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.swipeLeft;
+import static android.support.test.espresso.action.ViewActions.swipeRight;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.DrawerMatchers.isClosed;
+import static android.support.test.espresso.matcher.ViewMatchers.*;
+import static org.hamcrest.core.AllOf.allOf;
+import static org.junit.Assert.*;
+
+@RunWith(AndroidJUnit4.class)
+public class ScheduleParentFragmentTest {
+
+    @Rule
+    public final ActivityTestRule<EventShowcaseActivity> mActivityRule =
+            new ActivityTestRule<>(EventShowcaseActivity.class);
+
+    @Test
+    public void navigateInViewPager() {
+        Intent intent = new Intent();
+
+        intent.putExtra(EventPickingActivity.SELECTED_EVENT_ID, 2);
+        mActivityRule.launchActivity(intent);
+        onView(withId(R.id.drawer_layout))
+                .check(matches(isClosed(Gravity.LEFT)))
+                .perform(DrawerActions.open());
+
+        // Display Schedule Events
+        onView(withId(R.id.nav_view))
+                .perform(NavigationViewActions.navigateTo(R.id.nav_schedule));
+
+        SystemClock.sleep(800);
```

Comment:
Do we really need those sleeps? Aren't the animations disabled?

Reply:
Even if the animation should be disabled, there were some cases where the test was failing on travis and that did the trick so I prefer to keep it

Reply:
Let's say it's good enough.